include/odp/api/spec/schedule_types.h

```diff
@@ -159,6 +165,107 @@ typedef	struct odp_schedule_param_t {
 	uint32_t lock_count;
 } odp_schedule_param_t;
 
+/**
+ * Scheduler capabilities
+ */
+typedef struct odp_schedule_capability_t {
+	/** Maximum number of ordered locks per queue */
+	uint32_t max_ordered_locks;
+
+	/** Maximum number of scheduling groups */
+	unsigned max_groups;
+
+	/** Number of scheduling priorities */
+	unsigned prios;
```

Comment:
It would be a good opportunity to change this to max_prios. So, that the new config struct can have a num_prios field, which application may use to tell how many priorities are actually needed.

Also it would be good to change these "unsigned" types to uint32_t. At least some time back, checkpatch did not allow any more "unsigned" but "unsigned int" should be used. uint32_t is commonly same type as "unsigned", so it should not cause harm to applications if we change to uint32_t.

Reply:
ack